docs/resource-sharing/os-backfill-containers.md

```diff
@@ -74,93 +82,71 @@ docker run -it --rm --user osg  \
        opensciencegrid/osgvo-docker-pilot:release
 ```
 
-Replace `/path/to/token` with the location you saved the token obtained from the OSP administrators.
+Replace `/path/to/token` with the location you saved the token obtained from the OSPool Token Registry.
 Privileged mode (`--privileged`) requested in the above `docker run` allows the container
 to mount [CVMFS using cvmfsexec](#adding-cvmfs-using-cvmfsexec) and invoke `singularity` for user jobs.
-Singularity allows OSP users to use their own container for their job (e.g., a common use case for GPU jobs).
+Singularity allows OSPool users to use their own container for their job (e.g., a common use case for GPU jobs).
 
+Optional Configuration
+----------------------
 
-CVMFS
------
+### (Recommended) CVMFS
 
 [CernVM-FS](https://cernvm.cern.ch/fs/) (CVMFS) is a read-only remote filesystem that many OSG jobs depend on for software and data.
 Supporting CVMFS inside your container will greatly increase the types of OSG jobs you can run.
 
 There are two methods for making CVMFS available in your container: [enabling cvmfsexec](#adding-cvmfs-using-cvmfsexec),
-or [bind-mounting CVMFS from the host](#adding-cvmfs-via-bind-mount).
-Bind-mounting CVMFS will require CVMFS to be installed on the host first,
+or [bind mounting CVMFS from the host](#adding-cvmfs-via-bind-mount).
+Bind mounting CVMFS will require CVMFS to be installed on the host first,
 but the container will need fewer privileges.
 
 
-### Adding CVMFS using cvmfsexec
-
-[cvmfsexec](https://github.com/CVMFS/cvmfsexec#readme) is a tool that can be used to mount CVMFS inside the container
-without requiring CVMFS to be installed on the host.
-
-To enable cvmfsexec, specify a space-separated list of repos in the `CVMFSEXEC_REPOS` environment variable.
-We recommend the following repos:
--   `oasis.opensciencegrid.org`
--   `singularity.opensciencegrid.org`
-
-cvmfsexec has the following system requirements:
+#### cvmfsexec
 
--   On EL7, you must have kernel version >= 3.10.0-1127 (run `uname -vr` to check), and user namespaces enabled.
-    See step 1 in the
-    [Singularity Install document](https://opensciencegrid.org/docs/worker-node/install-singularity/#enabling-unprivileged-singularity)
-    for details.
+!!! info "`cvmfsexec` System Requirements"
```

Comment:
Not a big fan of the backticks around "cvmfsexec". Backticks should be reserved for command lines and symbols whereas here it's being used generically to refer to the entire product.

Reply:
Fixed